packages/nodes-base/nodes/Code/Code.node.ts

```diff
@@ -13,6 +13,8 @@ import { PythonSandbox } from './PythonSandbox';
 import { getSandboxContext } from './Sandbox';
 import { standardizeOutput } from './utils';
 
+const { CODE_DISABLE_STDOUT } = process.env;
```

Comment:
Any reason we don't add this to the environment schema?

Reply:
none of the other Code node related variables are currently in the config schema. mostly because `config` is still a part of the `cli` package, and we currently have no way to pass in config to the nodes code. We should eventually extract config into it's own package, so that we can stop using `process.env` in `core` and `nodes-base`, but that's out of the scope of this PR.